corehq/apps/userreports/tests/test_view.py

```diff
@@ -716,3 +716,142 @@ def test_subscribe_unsuccessful_with_missing_params(self):
             request.content.decode("utf-8"),
             "Missing parameters: client_id, token_url",
         )
+
+
+class TestUnsubscribeFromDataSource(TestCase):
+
+    urlname = "unsubscribe_from_configurable_data_source"
+    domain = "test-domain"
+    client_id = "client_id"
+    USERNAME = "username"
```

Comment:
There is nothing wrong with this code, but there are things that I'm curious about:

* "USERNAME" is capitalized but "domain" and "client_id" are lowercase. Was this a deliberate choice?
* "USERNAME", "domain" and "client_id" are defined because they are used more than once. But "urlname" is only used once. Is there a reason you assigned it to a class property instead of hard-coding it like the API username and the passwords?